geospaas_harvesting/crawlers.py

```diff
@@ -231,12 +250,44 @@ def _get_links(cls, html):
         return parser.links
 
 
+class PODAACCrawler(WebDirectoryCrawler):
+    """
+    Crawler for harvesting the data of PODAAC
+    """
+    LOGGER = logging.getLogger(__name__ + '.PODAACCrawler')
+    FOLDERS_SUFFIXES = ('/contents.html',)
+    FILES_SUFFIXES = ('.nc', '.nc.gz')
+    EXCLUDE = ['?']
+
+    def get_download_url(self, resource_url):
+        return resource_url
+
+
+class OpenDAPCrawler(WebDirectoryCrawler):
+    """
+    Crawler for harvesting the data which are provided by OpenDAP
+    """
+    LOGGER = logging.getLogger(__name__ + '.OpenDAPCrawler')
+    FOLDERS_SUFFIXES = ('/catalog.html',)
+    FILES_SUFFIXES = ('.nc',)
+    # We exclude "EASE-Grid map projections" and "southern hemispheres" from harvesting provess
+    EXCLUDE = ['/thredds/', 'http', '_sh_polstere', 'ease', ]
+
+    def get_download_url(self, resource_url):
+        links = self._get_links(self._http_get(resource_url))
+        for link in links:
+            if "dodsC" in link:
+                if not link.endswith(".html"):
+                    raise ValueError('The link as the result of crawler must be ended with ".html"')
```

Comment:
I have changed it to 
```python
self.LOGGER.info('The link as the result of crawler for "%s" must be ended with ".html".Failed to create downloable form',resource_url)
```

Reply:
That would rather be a warning.
You also need to explicitly return `None` here and act accordingly in `_explore_page()`, otherwise you will end up adding `None` in the list of links to return.

Reply:
Done in the next commit